man/md2man-all.sh

```diff
@@ -9,14 +9,19 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 	pwd
 }
 
-for FILE in *.md; do
-	base="$(basename "$FILE")"
-	name="${base%.md}"
-	num="${name##*.}"
-	if [ -z "$num" -o "$name" = "$num" ]; then
-		# skip files that aren't of the format xxxx.N.md (like README.md)
-		continue
-	fi
-	mkdir -p "./man${num}"
-	go-md2man -in "$FILE" -out "./man${num}/${name}"
-done
+if ( which go-md2man &>/dev/null ); then
```

Comment:
Sorry for not responding. Can you please simplify this change by just adding 

```
if ! ( which go-md2man &>/dev/null ); then
	echo "To install man pages, please install 'go-md2man'."
	exit 0
fi
```

And then leave the rest of the script unchanged?

Reply:
OK I got it !! :smile: